roadmap/todo.lean

```diff
@@ -0,0 +1,2 @@
+-- Axiom used to skip proofs in formal roadmaps.
+axiom todo {p : Prop} : p
```

Comment:
Do we want a corresponding tactic, that is simply a wrapper for `exact todo`?

Reply:
I added this tactic.